### PR TITLE
Build broken with latest bcc 

### PR DESCRIPTION
**Pull request by williangaspar**
The source of the problem was how BCC loads the `bpf.h` file. It used to get it from a project folder called `compat`, but now it's loading it from `/usr/include/linux/bpf.h`.

If the user tries to run `cgroup` in an old kernel there will an error on `semantic_nalyser.cpp`.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -58,6 +58,13 @@ set(CMAKE_REQUIRED_LIBRARIES bcc)
 check_symbol_exists(bcc_prog_load "${LIBBCC_INCLUDE_DIRS}/libbpf.h" HAVE_BCC_PROG_LOAD)
 check_symbol_exists(bcc_create_map "${LIBBCC_INCLUDE_DIRS}/libbpf.h" HAVE_BCC_CREATE_MAP)
 
+include(CheckTypeSize)
+set(CMAKE_EXTRA_INCLUDE_FILES linux/bpf.h)
+# This will set HAVE_GET_CURRENT_CGROUP_ID to TRUE or FALSE
+check_type_size("BPF_FUNC_get_current_cgroup_id" GET_CURRENT_CGROUP_ID LANGUAGE C)
+message("${HAVE_GET_CURRENT_CGROUP_ID}")
```
Comment (mmarchini) [inline CMakeLists.txt]:
```suggestion

```

File: src/ast/semantic_analyser.cpp [hunk 2]
```diff
@@ -59,6 +59,12 @@ void SemanticAnalyser::visit(Builtin &builtin)
       builtin.ident == "ctx" ||
       builtin.ident == "retval") {
     builtin.type = SizedType(Type::integer, 8);
+    if (builtin.ident == "cgroup") {
+      #ifndef HAVE_GET_CURRENT_CGROUP_ID
+        std::cerr << "BPF_FUNC_get_current_group_id is not available for your kernel version" << std::endl;
```
Comment (mmarchini) [inline src/ast/semantic_analyser.cpp]:
```suggestion
        err_ << "BPF_FUNC_get_current_group_id is not available for your kernel version" << std::endl;
```

File: src/ast/semantic_analyser.cpp [hunk 3]
```diff
@@ -59,6 +59,12 @@ void SemanticAnalyser::visit(Builtin &builtin)
...
+        abort();
```
Comment (mmarchini) [inline src/ast/semantic_analyser.cpp]:
```suggestion

```

**Comment by brendangregg** [comment]
Makes sense, thanks!

**Pull request merged**